modules/kubernetes-addons/crossplane/locals.tf

```
@@ -21,9 +21,23 @@ locals {
     operating-system = "linux"
   })]
 
-  aws_provider_sa        = "aws-provider"
-  jet_aws_provider_sa    = "jet-aws-provider"
-  kubernetes_provider_sa = try(var.helm_config.service_account, "kubernetes-provider")
-  aws_current_account_id = var.account_id
-  aws_current_partition  = var.aws_partition
+  aws_provider = merge({
+    provider_aws_version     = "v0.34.0"
+    additional_irsa_policies = ["arn:aws:iam::aws:policy/AdministratorAccess"]
```

so that this works with other non-commercial regions
```suggestion
    additional_irsa_policies = ["arn:${var.addon_context.aws_partition_id}:iam::aws:policy/AdministratorAccess"]
```

---

I took the policy from the example repo, this is what they were passing in 
https://github.com/awslabs/crossplane-on-eks/blob/main/bootstrap/terraform/main.tf#L156